docs/user-guide/src/ipam/ipam.md

```diff
@@ -0,0 +1,52 @@
+# IPAM (IP Address Manager)
+
+The IPAM project provides a controller to manage static IP address allocations
+in [Cluster API Provider Metal3](https://github.com/metal3-io/cluster-api-provider-metal3/).
+
+In CAMP3, the Network Data need to be passed to ironic through the BMH (Bare
+Metal Host). CAPI addresses the deployment of Kubernetes clusters and nodes, using the
+Kubernetes API. As such, it uses objects such as Machine Deployments (similar
+to deployments for pods) that takes care of creating the requested number of
+machines, based on templates. The replicas can be increased by the user,
+triggering the creation of new machines based on the provided templates.
+Considering the Kubeadm Control Plane and machine deployment features in
+Cluster API, it is not possible to provide static IP addresses
+for each machine before the actual deployments.
+
+In addition, all the resources from the source cluster must support the CAPI
+pivoting, i.e. being copied and recreated in the target cluster. This means
+that all objects must contain all needed information in their spec field to
+recreate the status in the target cluster without losing information. All
+objects must, through a tree of owner references, be attached to the cluster
+object, for the pivoting to proceed properly.
```

Comment:
Here also I am not sure what's the relationship of this context TBH

Reply:
Removed.